cli/shared/src/main/scala/com/lightbend/rp/reactivecli/process/dockercred.scala

```diff
@@ -64,6 +66,22 @@ object dockercred extends LazyLogging {
     }
   }
 
+  // Returns seq of tuples (server, username, helper) merged from multiple helpers
```

Comment:
I wonder if this might be clearer with a Future.sequence(ks.flatMap(...)), and no recursion, but I'd have to try, and a bit short on time just now. This is also good. Thanks!